client/my-sites/stats/stats-subscribers-page/subsriber-highlights.jsx

```diff
@@ -0,0 +1,97 @@
+import { HighlightCard, Popover } from '@automattic/components';
+import { Icon, info } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import '../wordads/highlights-section.scss';
+
+function getSubscriberHighlights() {
+	const highlights = [
+		{
+			id: 1,
+			heading: 'Total email subscribers',
+			count: 1410,
+			previousCount: 1080,
+		},
+		{
+			id: 2,
+			heading: 'Free email subscribers',
+			count: 1340,
+			previousCount: 1034,
+		},
+		{
+			id: 3,
+			heading: 'Paid email subscribers',
+			count: 70,
+			previousCount: 46,
+		},
+		{
+			id: 4,
+			heading: 'WordPress.com subscribers',
+			count: 322,
+			previousCount: 326,
+		},
+	];
+	return highlights;
+}
+
+function SubscriberHighlightsHeader() {
+	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
+	const infoReferenceElement = useRef( null );
+	const localizedTitle = translate( 'All time stats', {
+		comment: 'Heading for Subscribers page highlights section',
+	} );
+	const showNotices = true;
+	const tooltipContent = <p>Tooltip content goes here.</p>;
+	return (
+		<h1 className="highlight-cards-heading">
+			{ localizedTitle }{ ' ' }
+			{ showNotices && (
+				<>
+					<span
+						className="info-wrapper"
+						ref={ infoReferenceElement }
+						onMouseEnter={ () => setTooltipVisible( true ) }
+						onMouseLeave={ () => setTooltipVisible( false ) }
+					>
+						<Icon className="info-icon" icon={ info } />
+					</span>
+					<Popover
+						className="tooltip tooltip--darker tooltip-wordads highlight-card-tooltip"
+						isVisible={ isTooltipVisible }
+						position="bottom right"
+						context={ infoReferenceElement.current }
+					>
+						<div className="highlight-card-tooltip-content">{ tooltipContent }</div>
+					</Popover>
+				</>
+			) }
+		</h1>
+	);
+}
+
+function SubscriberHighlightsListing() {
+	const highlights = getSubscriberHighlights();
+
+	return (
+		<div className="highlight-cards-list">
+			{ highlights.map( ( highlight ) => (
+				<HighlightCard
+					key={ highlight.id }
+					heading={ highlight.heading }
+					count={ highlight.count }
+					previousCount={ highlight.previousCount }
+					showValueTooltip
+				/>
+			) ) }
+		</div>
+	);
+}
+
+export default function SubscriberHighlights() {
+	return (
+		<div className="highlight-cards wordads has-odyssey-stats-bg-color">
```

Comment:
Similar to what Jason mentioned for the heading, this element could use a class specific to subscribers highlights because it's the parent.

It looks like `wordads has-odyssey-stats-bg-color`are redundant.

Reply:
The `wordads` is necessary there to get the info icon vertically aligned and properly colored. I've updated it to `subscribers-page` now that the CSS lives in a new file.

As for `has-odyssey-stats-bg-color`, I believe it's only defined when running in Odyssey stats. It's still defined in the code and present on the other highlight sections on the Traffic and Insights page. I'll take a closer look at this and open a follow-up PR if needed though.

Reply:
Note that `.subscribers-page` may lead to styling leaks like `.promo-card` ([issue](https://github.com/Automattic/wp-calypso/issues/76032)). I recommend we use something more specific to this component, like `.stats-subscriber-highlights`.